src/test/resources/SubscriptionObjectForJenkins.json

```diff
@@ -0,0 +1,39 @@
+{
+        "subscriptionName": "Subscription_1",
+        "userName" : "DEF",
+        "repeat": false,
+        "created": "data-time",
+        "notificationType": "REST_POST",
+        "notificationMeta": "https://fem101-eiffel039.lmera.ericsson.se:8443/jenkins/job/test_params/build",
```

Comment:
Change so we don't have ericsson addresses on github. Example: "https://jenkins.domain.com:8100/jenkins/job/test_params/build"